charts/nri-prometheus/Chart.yaml

```diff
@@ -1,19 +1,36 @@
-apiVersion: v1
-appVersion: 2.14.0
-description: A Helm chart to deploy the New Relic Prometheus OpenMetrics integration
+apiVersion: v2
+
 name: nri-prometheus
-version: 1.14.0
-engine: gotpl
-home: https://github.com/newrelic/nri-prometheus
-icon: https://newrelic.com/assets/newrelic/source/NewRelic-logo-square.svg
+description: A Helm chart to deploy the New Relic Prometheus OpenMetrics integration
+
+home: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations/
+icon: https://newrelic.com/themes/custom/curio/assets/mediakit/new_relic_logo_vertical.svg
+sources:
+  - https://github.com/newrelic/nri-prometheus
+
+version: 2.0.0
```

Comment:
just realized that there this should be just a minor bump, I will change it when addressing more changes request from reviews